TRADEMARK-COMMITTEE.md

```diff
@@ -0,0 +1,145 @@
+conformance, and scope.
+
+Decisions are made in meetings when a quorum of the members are present and may
+pass with at least half the members of the committee supporting it. Vetoes by
+absent committee members must be presented within 5 business days of a published
+decision.
+
+Quorum is considered reached when at least half of the members are present.
+Quorum may be established over email or electronic chat systems as well as via
+video or in-person meetings.
+
+In case of extended absence or removal from the project for a code of conduct
+violation, the organization of the absent member may appoint a single
+replacement delegate from the same company during the absence.
+
+## Getting in touch
+
+There are two ways to raise issues to the steering committee for decision:
+
+1. Emailing the steering committee at
+   [knative-trademark@googlegroups.com](mailto:knative-trademark@googlegroups.com).
+   This is a private discussion list to which all members of the committee have
+   access.
+2. Open an issue on a knative repository and indicate that you would like
+   attention from the trademark committee.
+
+## Composition
+
+The Trademark Committee is composed of company representatives from companies
+which have substantially contributed to Knative over the last year, as well as
+the trademark holder. These representatives shall be confirmed by the Knative
+Steering Committee; companies which have a currently-unfilled seat on the
+Trademark Committee may veto decisions made while the seat was empty as if those
+decisions had been made at the time of confirmation.
+
+Company representatives serving on the Trademark Committee may individually
+resign or be removed from their seats by their affiliated company with at least
+5 days notice.
+
+For the 2020 year, one seat is assigned to each of Google, IBM/RedHat, and
+VMware. The committee shall review this composition and the selection rules at
+least annually in October of each year (to be aligned with the Steering
+elections process).
+
+### Maintaining the Trademark Committee
+
+The Trademark Committee membership is refreshed annually on the same timeline as
+the Steering Committee elections (during the month of October). The Trademark
+Committee's size is unbounded and may grow or shrink depending on the activity
+in the project in the year prior to each refresh.
+
+Membership in the committee is determined by counting the
+[easily countable contributions over the last year](https://knative.teststats.cncf.io/d/5/companies-table?orgId=1&var-period_name=Last%20year&var-metric=contributions).
+Each company which has contributed at least 15% of the total project
+contributions will be eligible to select one representative to the committee.
+Not all contributions are easy to count and many significant contributions are
+hard to count. There will be an exception process that allows vendors to
+articulate why they should receive consideration for membership on the Trademark
```

Comment:
s/There will be an exception process that allows vendors ... /Vendors may petition the Trademark Committee to receive consideration/

Reply:
Done.